website/content/en/docs/cli/operator-sdk_new.md

```diff
@@ -66,17 +66,18 @@ operator-sdk new <project-name> [flags]
 ### Options
 
 ```
-      --api-version string          Kubernetes apiVersion and has a format of $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1) - used with "ansible" or "helm" types
-      --crd-version string          CRD version to generate (Only used for --type=ansible|helm) (default "v1beta1")
+      --api-version string          Kubernetes apiVersion and hasa format of $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1)
```

Comment:
typo: hasa -> has a

Reply:
This is being tricky with auto-generation of website/docs/cli. I cannot change the content directly in `website/docs/cli`, and the content itself is not there anymore in `cmd/operator-sdk/new/cmd.go` , as we have moved our flags to `internal/flags/apiflags`.  I need to work with @asmacdo to see how to generate these docs fresh.